repositories/decisions_repository.go

```diff
@@ -393,6 +401,7 @@ func (repo *DecisionRepositoryImpl) DecisionsOfScheduledExecution(
 		ctx,
 		exec,
 		selectDecisions().
+			Where(squirrel.Eq{"org_id": organizationId}).
```

Comment:
First level of fix for permissions when querying decisions by scheduled_execution_id. It's still not perfect because I'd rather this permission is enforced in the usecase, as in the rest of our code. However, because this function uses a channel of decisions that are streamed directly to the writer higher up the stack, it's a bit more complicated to do so here (we should probably add an intermediate layer in the usecase where we read the repo's channel and send to another channel higher up.
Anyway, this is the minimum change so that one cannot query another org's scheduled ex decisions.